drivers/volume/kdmp/kdmp.go

```diff
@@ -765,6 +787,12 @@ func (k *kdmp) StartRestore(
 		dataExport.Annotations[utils.SkipResourceAnnotation] = "true"
 		dataExport.Annotations[utils.BackupObjectUIDKey] = backupUID
 		dataExport.Annotations[pvcUIDKey] = bkpvInfo.PersistentVolumeClaimUID
+		logrus.Info("enableKopiaExecutorDebugMode calling")
+		if enableKopiaExecutorDebugMode() {
+			logrus.Info("enableKopiaExecutorDebugMode able to find the debug mode present in kdmp-config cm")
```

Comment:
Redundant Info log. Its already getting printed in check function.. hence this should be removed to reduce logfile payload.

Reply:
Yup, sorry about this, I forgot to remove log messages while raising the PR